background_scripts/completion_search.coffee

```diff
@@ -42,7 +42,7 @@ CompletionSearch =
 
   # The amount of time to wait for new requests before launching the current request (for example, if the user
   # is still typing).
-  delay: 100
+  delay: 0
```

Comment:
I know the intention was good, but a 100ms delay does nothing to reduce the number of requests, it just leaves us with 50ms to do all our processing and requests rather than 150ms.

If we're going to rate limit, let's do it in one place (the frontend) and do it properly (longer than 150ms).